pkg/moov/transfer_api.go

```diff
@@ -46,23 +37,57 @@ func (c Client) CreateTransfer(ctx context.Context, transfer CreateTransfer, opt
 		callArgs = append(callArgs, opt(builder))
 	}
 
-	resp, err := c.CallHttp(ctx, Endpoint(http.MethodPost, pathTransfers), callArgs...)
+	return CreateTransferBuilder{
+		client:   c,
+		ctx:      ctx,
+		endpoint: Endpoint(http.MethodPost, pathTransfers),
+		callArgs: callArgs,
+	}
+}
+
+type CreateTransferBuilder struct {
+	client   Client
+	ctx      context.Context
+	endpoint EndpointArg
+	callArgs []callArg
+}
+
+// Started kicks off the transfers request and doesn't wait beyond the initial response kicking off the transfer
+func (r CreateTransferBuilder) Started() (*TransferStarted, error) {
+	resp, err := r.client.CallHttp(r.ctx, r.endpoint, r.callArgs...)
+	if err != nil {
+		return nil, err
+	}
+
+	switch resp.Status() {
+	case StatusCompleted:
+		st, err := UnmarshalObjectResponse[TransferStarted](resp)
+		return st, err
+	case StatusStateConflict:
+		return nil, errors.Join(ErrXIdempotencyKey, resp)
+	default:
+		return nil, resp
+	}
+}
+
+// Starts a transfer request and then waits for the approval or refusal of the rail before returning the result.
+// This returns 3 results
+// - 1st being a transfer that waited until we got a success or failure from the rail.
+// - 2nd being a transfer that was started but didn't finish before timing out. This response can be retried with the same idempotency key. You may also void or refund using the data from the response.
+// - 3rd being an error in running the transfer which is a terminal state.
```

Comment:
```suggestion
// The 3 return values are:
// 1) A full transfer with rail-specific details as a result of waiting for the response from the rail.
// 2) A transfer that started but the request timed out waiting for a response from the rail.
// 3) An error attempting to create the transfer.
```


> This response can be retried with the same idempotency key. 

I'm not I understand these lines. If they retry the transfer with the same idempotency key, they'll get a 409.

> You may also void or refund using the data from the response.

I would recommend omitting this too. This is too specific to card-acquiring and doesn't apply to all rails